src/entries/popup/hooks/useHiddenAssets.ts

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from 'react';
+
+import { useHiddenAssetStore } from '~/core/state/hiddenAssets/hiddenAssets';
+import { ParsedUserAsset } from '~/core/types/assets';
+import { SearchAsset } from '~/core/types/search';
+
+export const useHiddenAssets = () => {
```

Comment:
why are these methods not in the store itself?

Reply:
Yeah. We could do it from store itself, but there would be a lot of duplicate functions since we use the same `${asset.address}-${asset.chainId}` string to add and remove hidden assets + `isHidden` is used a lot so didn't want to duplicate that as well. 

Keeping this in a hook would just make things simpler. Lmk what you think 🙏

Reply:
I mean something like 
```ts
export const hiddenAssets = createStore<HiddenAssetState>(
  (set, get) => ({
    hiddenAssets: [],
    addHiddenAsset: (asset: { chainId: ChainId, address: Address }) => {
      const { hiddenAssets } = get();
      const uniqueId = `${asset.address}-${asset.chainId}`
      set({ hiddenAssets: [...hiddenAssets, uniqueId] });
    },
    isHidden: (asset) => {
      const { hiddenAssets } = get();
      const uniqueId = `${asset.address}-${asset.chainId}`
      return hiddenAssets.includes(uniqueId)
    }
```
so whats outside the store don't need to know about how it stores its data, just pass an asset and it handles the rest

Reply:
and basically rename `useHiddenAssetStore` -> `useHiddenAssets` I think would work the same

Reply:
Ah ok. Yeah makes sense now, let me do that. Thanks 🙏